test/spirv_report.spt

```diff
@@ -1,5 +1,5 @@
 ; RUN: llvm-spirv %s -to-binary -o %t.spv
-; RUN: llvm-spirv --spirv-print-report %t.spv | FileCheck %s --check-prefix=CHECK-DAG
```

Comment:
What will be reported if an extension is missing in spirv-ext? Can we add a CHECK for that?

Thanks

Reply:
Exactly the same like in other tests which covers situation with missing extensions. We already have such tests. For more detailed response please look on the answer below.